server/controllers/temp-outfitter.es6

```diff
@@ -102,7 +104,7 @@ let translateFromDatabaseToClient = input => {
     createdAt: input.createdAt,
     district: input.district,
     forest: input.forest,
-    reasonForReturn: input.reasonForReturn || undefined,
```

Comment:
On the frontend the naming of this component seems to be `reasonOrCancel`. I wonder if it makes sense to call it something `messageToApplicant` across the frontend / server.